lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -113,18 +99,31 @@ fields:
   cardinality: 0..*
   type: BackboneElement
   description: Adverse Reaction Events linked to exposure to substance
-  url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.code
+  url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.code
   children:
 
   - name: manifestation
     required: 'Yes'
     cardinality: 1..*
     type: CodeableConcept
     description: Clinical symptoms/signs associated with the Event
-    url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.manifestation
+    url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.manifestation
     binding:
       description: Clinical symptoms and/or signs that are observed or associated with the adverse reaction event.
       terminology:
       - display: SNOMED CT Clinical Findings
         system: http://snomed.info/sct
         info_link: http://hl7.org/fhir/r4/valueset-clinical-findings.html
+
+  - name: severity
+    required: 'No'
+    cardinality: 0..1
+    type: code
+    description: mild | moderate | severe (of event as a whole)
+    url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.severity
+    binding:
+      description: Clinical assessment of the severity of a reaction event as a whole.
+      terminology:
+      - display: AllergyIntoleranceSeverity
+        system: http://hl7.org/fhir/reaction-event-severity
+        info_link: http://hl7.org/fhir/R4/valueset-reaction-event-severity.html
```

Comment:
Same here. Change the info_link to a secure one.

Reply:
Seems like you need to update all the info_link tags to a secured link.

Reply:
updated